spec/factories/education_benefits_claims.rb

```diff
@@ -155,5 +155,13 @@
       }.to_json)
       form_type('1990n')
     end
+
+    factory :education_benefits_claim_5495 do
+      form({
+        benefit: 'chapter35',
```

Comment:
i can make a pr in the schema repo to make the applicant name required for all the schemas